CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## vNext
 ------------------
+## vNext (TBD)
```

Comment:
The changelog didn't cherry-pick nicely, but I'll fix that when preparing the release.